docs/developer/developer-mode-installation.md

```diff
@@ -0,0 +1,144 @@
+
+- The Node must pass the [host-check](https://raw.githubusercontent.com/harvester/harvester/master/hack/host-check.sh)
+- Helm 3 and Git are installed on your local machine.
+
+## Installation
+
+For development purpose, Harvester can be installed on a [RKE2](https://docs.rke2.io/) cluster by using the [Helm](https://helm.sh/) CLI.
+
+Please refer to the Harvester [Helm chart](https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/README.md) for more details on installing and configuring the Helm chart.
+
+1. Add RKE2 config.
+    ```bash
+    sudo mkdir -p /etc/rancher/rke2
+    cat <<EOF | sudo tee /etc/rancher/rke2/config.yaml
+    disable:
+    - rke2-snapshot-controller
+    - rke2-snapshot-controller-crd
+    - rke2-snapshot-validation-webhook
+    node-label:
+    - harvesterhci.io/managed=true
+    token: token
+    cni:
+    - multus
+    - canal
+    EOF
+    ```
+
+1. Install RKE2.
+    ```bash
+    curl -sfL https://get.rke2.io | sudo sh -
+    sudo systemctl enable rke2-server.service
+    sudo systemctl start rke2-server.service
+    ```
+
+1. Get kubeconfig.
+    ```bash
+    sudo chmod 644 /etc/rancher/rke2/rke2.yaml
+    mkdir -p ~/.kube
+    cp /etc/rancher/rke2/rke2.yaml ~/.kube/config
+    ```
+
+1. Install system-upgrade-controller. The component installs `upgrade.cattle.io/v1` CRDs.
+    ```bash
+    kubectl apply -f https://github.com/rancher/system-upgrade-controller/releases/download/v0.13.1/system-upgrade-controller.yaml
+    ```
+
+1. Create `cattle-system` namespace.
+    ```bash
+    kubectl create ns cattle-system
+    ```
+
+1. Add Rancher repo.
+    ```bash
+    helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+    ```
+
+1. Install rancher v2.7.5 chart.
+    ```bash
+    helm install rancher rancher-latest/rancher \
+    --namespace cattle-system \
+    --set tls=external \
+    --set rancherImagePullPolicy=IfNotPresent \
+    --set rancherImage=rancher/rancher \
+    --set rancherImageTag=v2.7.5 \
+    --set noDefaultAdmin=false \
+    --set features="multi-cluster-management=false\,multi-cluster-management-agent=false" \
+    --set useBundledSystemChart=true \
+    --set bootstrapPassword=admin
+    ```
+
+1. Clone rancher/charts repository.
+    ```bash
+    git clone https://github.com/rancher/charts -b dev-v2.7
+    ```
+
+1. Install rancher-montering-crd chart.
+    ```bash
+    helm install rancher-monitoring-crd ./charts/charts/rancher-monitoring-crd/102.0.2+up40.1.2/
+    ```
+
+1. Create `harvester-system` namespace.
+    ```bash
+    kubectl create ns harvester-system
+    ```
+
+1. Clone harvester/harvester repository.
+    ```bash
+    git clone https://github.com/harvester/harvester.git
+    ```
+
+1. Install harvester-crd chart.
+    ```bash
+    helm install harvester-crd ./harvester/deploy/charts/harvester-crd --namespace harvester-system
+    ```
+
+1. Install harvester chart. Use static mode kube-vip for developer mode installation.
```

Comment:
```suggestion
1. Install the Harvester chart using kube-vip running on a static pod.
```

Reply:
"static mode" does not seem to be an official term (I cannot find it in the kube-vip docs). Is the revised wording correct?